Extensions/Cosmos/Cosmos.DataTransfer.CosmosExtension/CosmosExtensionServices.cs

```diff
@@ -36,7 +38,17 @@ public static CosmosClient CreateClient(CosmosSettingsBase settings, string disp
             if (settings.UseRbacAuth)
             {
                 TokenCredential tokenCredential = new DefaultAzureCredential(includeInteractiveCredentials: settings.EnableInteractiveCredentials);
-                cosmosClient = new CosmosClient(settings.AccountEndpoint, tokenCredential, clientOptions);
+
+                if(settings.InitClientEncryption)
+                {
+                    var keyResolver = new KeyResolver(tokenCredential);
+                    cosmosClient = new CosmosClient(settings.AccountEndpoint, tokenCredential, clientOptions)
+                        .WithEncryption(keyResolver, KeyEncryptionKeyResolverName.AzureKeyVault);
```

Comment:
The Readme for the cosmos extension is missing the key vault information for the client-side encryption information needed here. Can you add?

Reply:
Thanks, @markjbrown.
Readme updated.